### PR TITLE
standardizing logging V1

### PR DESCRIPTION
**Pull request by goodenou**
I have modified the code in src/decisionengine_modules/*py, src/decisionengine_modules/AWS, src/decisionengine_modules/GCE, src/decisionengine_modules/graphite, and src/decisionengine_modules/glideinwms to be consistent with the logging standards.

### CODE REVIEW

File: src/decisionengine_modules/glideinwms/transforms/glidein_requests.py [hunk 1]
```diff
@@ -170,8 +167,8 @@ def merge_requests(self, manifests, group_manifests):
             m_keys = set(manifests.keys())
             g_keys = set(group_manifests.keys())
             if m_keys != g_keys:
-                raise RuntimeError(
-                    'Mismatch in manifest keys: %s, %s' % (m_keys, g_keys))
+                self.logger.exception(f"Mismatch in manifest keys: {m_keys}, {g_keys}")
+                raise RuntimeError()
```
Comment (mambelli) [inline src/decisionengine_modules/glideinwms/transforms/glidein_requests.py:171]: Here and few lines below the message is all in the logging instruction and nothing in the exception.
Maybe something should be also in the exception that is being propagated up

File: src/decisionengine_modules/glideinwms/transforms/glidein_requests.py [hunk 2]
```diff
@@ -181,14 +178,17 @@ def merge_requests(self, manifests, group_manifests):
 
     def read_fe_config(self):
         if not os.path.isfile(self.de_frontend_configfile):
-            raise RuntimeError(
+            self.logger.exception(
                 'Error reading Frontend config for DE %s. '
                 'Run configure_gwms_frontend.py to generate one and after every change to the frontend configuration.' %
                 self.de_frontend_configfile)
+            raise RuntimeError()
```
Comment (mambelli) [inline src/decisionengine_modules/glideinwms/transforms/glidein_requests.py:185]: same

File: src/decisionengine_modules/glideinwms/transforms/glidein_requests.py [hunk 3]
```diff
@@ -181,14 +178,17 @@ def merge_requests(self, manifests, group_manifests):
...
+
         fe_cfg = eval(open(self.de_frontend_configfile, 'r').read())
+
         if not isinstance(fe_cfg, dict):
-            raise ValueError('Frontend config for DE in %s is invalid' %
-                             self.de_frontend_configfile)
+            self.logger.exception(f"Frontend config for DE in {self.de_frontend_configfile} is invalid")
+            raise ValueError()
```
Comment (mambelli) [inline src/decisionengine_modules/glideinwms/transforms/glidein_requests.py:191]: same (see above, line 171)

**Review by mambelli — APPROVED** [review]
I will approve and merge but please consider if the exceptions risen should have a message reinstated.

File: src/decisionengine_modules/glideinwms/transforms/glidein_requests.py (hunk 1 above)
Comment (goodenou) [reply src/decisionengine_modules/glideinwms/transforms/glidein_requests.py:171]: HI Marco. The logging.exception call automatically puts the stack trace into the message, so the exception is logged.

**Pull request merged**